docs-v2/integrations/all/checkout-com-sandbox/connect.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Checkout.com - Sandbox - How do I link my account?
+sidebarTitle: Checkout.com - Sandbox
+---
+
+
+
+# Overview
+
+To authenticate with Checkout, you will need:
+1. **Client ID** - A unique identifier for your application on Checkout.
+2. **Client Secret** - A key that grants full access to all your application and configurations.
+
+This guide will walk you through generating the Client ID and Client Secret within Checkout.
+
+
+## Prerequisites:
+- You should have a registered test account with Checkout. Create one on [Checkout Getting Started page](https://checkout.com/).
+- Permission to generate **API Keys**.
+
+#### Step 1: Creating an API key
```

Comment:
It seems to be named Access Key and maybe should be reflected in the name of the fields in the UI